clients/client-sdk/src/clients/identity.ts

```diff
@@ -21,16 +25,73 @@ import {
   Presentation,
   Timestamp,
   IPresentation,
-  ProofOptions
-} from '@iota/identity-wasm/web';
+  ProofOptions,
+  KeyType,
+  VerificationMethod,
+  MethodScope,
+  IClientConfig,
+  Network,
+  Client
+} from '@iota/identity-wasm/node';
+import addFormats from 'ajv-formats';
+import Ajv from 'ajv';
 
 export class IdentityClient extends BaseClient {
   private baseUrl: string;
+  private permaNode?: string;
+  private node?: string;
 
   constructor(config: ClientConfig) {
     super(config);
     this.baseUrl = this.useGatewayUrl ? this.isGatewayUrl!! : this.ssiBridgeUrl!!;
     this.baseUrl = this.baseUrl + `/api/${config.apiVersionSsiBridge}`;
+    this.permaNode = config?.permaNode;
+    this.node = config?.node;
+  }
+
+  /**
+   * Create a new decentralized digital identity (DID) locally. Identity DID document is signed and published to the ledger (IOTA Tangle). A digital identity can represent an individual, an organization or an object. The privateAuthKey controlling the identity is returned. It is recommended to securely (encrypt) store the privateAuthKey locally, since it is not stored on the APIs Bridge.
+   * @param username
+   * @param claimType
+   * @param claim
+   * @param hidden
+   * @returns
+   */
+  async createLocally(
+    username: string,
+    claimType = UserType.Person,
+    claim?: any,
+    hidden?: boolean
+  ) {
+    const existingUser = await this.search({ username: username });
+    if (existingUser.length === 0) {
+      const createIdentity: CreateIdentityBody = {
+        username,
+        hidden,
+        claim: {
+          ...claim,
+          type: claimType
+        }
+      };
+      const ajv = this.getAjv();
+      const validObject = ajv.validate(CreateIdentityBodySchema, createIdentity);
+      if (validObject) {
+        const identity = await this.createIdentity();
+        const user: User = {
+          ...createIdentity,
+          id: identity.id
+        };
+        await this.add(user);
```

Comment:
in this case the creator will always be empty. we could add it as param oder need to authenticate before...